upmc-enterprises/cda2fhir
# Upmcfhir 349 - REAL data type and Epic quantities.

#### What does this PR do?

Adds in support for the REAL data type to observations by turning them into valueQuantities. Will also optionally attempt to add unit if a SNOMED unit entry is present (this is an Epic-specific consideration).

#### Related JIRA tickets:

[UPMCFHIR-349](https://jira.amida.com/browse/UPMCFHIR-349)

#### How should this be manually tested?

Run the robust CCD sample through, and see if there are values for lab entries that previously had "REAL" data types.

#### Background/Context:

Noticed this is in the data, and should fix it.

#### New JIRA tickets for clinical review required?
